app/views/admin/unattached_promo_registrations/report.html.slim

```diff
@@ -13,12 +13,24 @@ scss:
         tr
           - if @reporting_interval != PromoRegistration::RUNNING_TOTAL
             th = reporting_interval_column_header(@reporting_interval)
+          - if @is_geo
+            th = "Country"
           - @event_types.each do |event_type|
             th = event_type_column_header(event_type)
         tbody
-          - events.each do |event|
-            tr
-              - if @reporting_interval != PromoRegistration::RUNNING_TOTAL
-                td = event.first
-              - @event_types.each do |event_type|
-                td = event.second[event_type]
\ No newline at end of file
+          - if @is_geo
+            - events.each do |date, events|
```

Comment:
IMO instead of `events` as the value, I think `events_on_date` sounds more intuitive.